services/service/client/utils.go

```diff
@@ -96,6 +97,72 @@ func StopRemoteService(ctx context.Context, conn *proxy.Conn, system pb.SystemTy
 	return nil
 }
 
+// StartManyRemoteServices is a helper function for starting a service on multiple remote targets.
+func StartManyRemoteServices(ctx context.Context, conn *proxy.Conn, system pb.SystemType, service string) error {
+	c := pb.NewServiceClientProxy(conn)
+	respChan, err := c.ActionOneMany(ctx, &pb.ActionRequest{
+		ServiceName: service,
+		SystemType:  system,
+		Action:      pb.Action_ACTION_START,
+	})
+
+	if err != nil {
+		return fmt.Errorf("can't start service %s - %v", service, err)
+	}
+
+	var errorList []error
+	for resp := range respChan {
+		if resp.Error != nil {
+			err := fmt.Errorf("can't start service on target %s: %w", resp.Target, resp.Error)
+			errorList = append(errorList, err)
+		}
+	}
+
+	if len(errorList) == 0 {
+		return nil
+	}
+
+	var errorMessages []string
+	for _, err := range errorList {
+		errorMessages = append(errorMessages, err.Error())
+	}
+
+	return fmt.Errorf("%s", strings.Join(errorMessages, "\n"))
+}
+
+// StopManyRemoteService is a helper function for stopping a service on multiple remote targets.
+func StopManyRemoteService(ctx context.Context, conn *proxy.Conn, system pb.SystemType, service string) error {
```

Comment:
Please keep the naming consistent and rename the function to `StopRemoteServiceMany()`